policy/modules/system/init.if

```diff
@@ -3263,6 +3263,24 @@ interface(`init_tcp_recvfrom_all_daemons',`
 	corenet_tcp_recvfrom_labeled($1, daemon)
 ')
 
+######################################
+## <summary>
+##	restart systemd units, for /run/systemd/transient/*
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`init_restart_units',`
+	gen_require(`
+		type init_var_run_t;
+	')
+
+	allow $1 init_var_run_t:service { start status stop };
```

Comment:
There already are interfaces for transient units (and a new type).

Reply:
ok